CRM/Custom/Form/Group.php

```diff
@@ -220,7 +220,7 @@ public function buildQuickForm() {
 
     foreach ($subTypes as $subType => $val) {
       $subTypeRelationshipTypes = CRM_Contact_BAO_Relationship::getContactRelationshipType(NULL, NULL, NULL, $val['parent'],
-        FALSE, 'label', TRUE, $subType
+        FALSE, 'label', TRUE, array($subType)
```

Comment:
Did you mean to do `(array) $subType` instead? This will place the variable into an array even if it is an array already.